dotnet/tests/MicroFocus.CafApi.QueryBuilder.Vqll.Builder.Internal.Tests/FilterWriterTest.cs

```diff
@@ -0,0 +1,3719 @@
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.VerifyNoOtherCalls();
+
+            FilterWriter.WriteToJsonArray(clause, _jsonBuilder.Object);
+        }
+
+        [Fact]
+        public void TestTransformFullTextNearSingleValueLiterals()
+        {
+            List<string> fieldSpecs = new List<string>() { "DATA_SUBJECT" };
+            LikeToken[] lhs = { LikeTokenFactory.StringLiteral("left") };
+            LikeToken[] rhs = { LikeTokenFactory.StringLiteral("right") };
+            FullTextFilter clause = FullTextFilterFactory.Near(lhs, rhs, 0);
+            Filter<string> filterClause = FilterFactory.FieldFullText(fieldSpecs, clause);
+
+            var sequence = new MockSequence();
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString("field-full-text"));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            foreach (string s in fieldSpecs)
+            {
+                _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString(s));
+            }
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString("near"));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteNumber(0));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString("'"));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString("left"));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteStartArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString("'"));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteString("right"));
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+            _jsonBuilder.InSequence(sequence).Setup(x => x.WriteEndArray());
+
+            _jsonBuilder.VerifyNoOtherCalls();
+
+            FilterWriter.WriteToJsonArray(filterClause, _jsonBuilder.Object);
+        }
+
+        [Fact]
+        public void TestTransformFullTextNearSpecialCharactersLiterals()
+        {
+            List<string> fieldSpecs = new List<string>() { "DATA_SUBJECT" };
```

Comment:
You don't need to repeat the type.  It can be fixed on either side:
```suggestion
            List<string> fieldSpecs = new() { "DATA_SUBJECT" };
```

```suggestion
            var fieldSpecs = new List<string>() { "DATA_SUBJECT" };
```

I'd have a slight preference for the first option.  This issue is throughout the file.